PlantPal/PlantPal/WelcomeScreen/WelcomeScreenView.swift

```diff
@@ -0,0 +1,88 @@
+//
+//  ContentView.swift
+//  PlantPal
+//
+//  Created by Marcin Dytko on 07/07/2024.
+//
+
+import SwiftUI
+// MARK: - VIEW
+
+struct WelcomeScreenView: View {
+    // MARK: - Properties
+    
+    @ObservedObject var viewModel: WelcomeScreenViewModel
+    
+    // MARK: - BODY
+    var body: some View {
+        NavigationStack {
+            MainStack {
+                
+                VStack {
+                    Image(.plant)
+                    
+                    Text("PlantPal AI")
+                        .font(.custom("Merriweather-Regular", size: 32))
+                        .padding(.bottom, 8)
+                    
+                    Text("Your pocket botanist")
+                        .font(.custom("Merriweather-Regular", size: 18))
+                        .padding(.bottom, 4)
+                    
+                    Divider()
+                        .padding(.bottom, 8)
+                    
+                    Text("""
+                    Discover the world of plants at your
+                    fingertips. Simply snap a photo, and
+                    PlantPal AI instantly identfies plant species
+                    """)
+                    .font(.custom("Merriweather-Light", size: 16))
+                    .multilineTextAlignment(.center)
+                    .padding(.bottom)
+                    
+                    Text("""
+                    We use OpenAI API. We do not charge any
+                    fees, so we need your ket to be able to
+                    recvie replies on your behalf
+                    
+                    We do not have access to your key!
+                    """)
+                    .font(.custom("Merriweather-Regular", size: 16))
+                    .multilineTextAlignment(.center)
+                    VStack(alignment: .leading) {
+                        
+                        Text("OpenAI Api key")
+                            .padding([.top, .horizontal], 8)
+                            .font(.custom("Merriweather-Regular", size: 12))
+                        
+                        TextField("sk-proj-***************************", text: $viewModel.apiKey)
```

Comment:
Tutaj musimy dać jeszcze modyfikator, który wyłączy autocapitalization wpisywanego tekstu i też warto wyłączyć automatyczne poprawianie wpisywanego tekstu.

Reply:
I wydzielmy też ten textfield i tekst "Key is invalid" do 
```suggestion
        private func buildInputView() {
           // TextField
           // Error Message
        }
```